public/pages/Channels/components/details/ChannelDetails.tsx

```diff
@@ -126,76 +144,111 @@ export function ChannelDetails(props: ChannelDetailsProps) {
+                    setSelected: (selected: any[]) => setChannel(selected[0]),
+                  });
+                } else {
+                  const newChannel = { ...channel, is_enabled: true };
+                  servicesContext.notificationService
+                    .updateConfig(channel.config_id, newChannel)
+                    .then(() => {
+                      coreContext.notifications.toasts.addSuccess(
+                        `Channel ${channel.name} successfully unmuted.`
+                      );
+                      setChannel(newChannel);
+                    });
+                }
+              }}
+            >
+              {channel.is_enabled ? 'Mute channel' : 'Unmute channel'}
+            </EuiSmallButton>
+          )}
+        </ModalConsumer>
+      )}
+    </EuiFlexItem>
+  </EuiFlexGroup>;
+  const rightControls = [
+    {
+      renderComponent: (
+        actionsAndMuteComponent
+      ),
+    },
+    {
+      renderComponent: (
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <EuiSmallButton
+            data-test-subj="send-test-message-button"
+            onClick={sendTestMessage}
+            style={{ marginLeft: '10px' }}
+            disabled={!channel?.is_enabled}
+          >
+            Send test message
+          </EuiSmallButton>
+        </div>
+      ),
+    },
+  ];
+
+  const badgeComponent = <EuiFlexItem grow={false} style={{ paddingBottom: 5 }}>
+    {channel?.is_enabled === undefined ? null : channel.is_enabled ? (
+      <EuiHealth color="success">Active</EuiHealth>
+    ) : (
+      <EuiHealth color="subdued">Muted</EuiHealth>
+    )}
+  </EuiFlexItem>;
+
+  const badgeControls = [
+    {
+      renderComponent: (
+        badgeComponent
+      ),
+    },
+  ];
+
   return (
     <>
-      <EuiGlobalToastList
-        toasts={toasts}
-        dismissToast={(removedToast) => {
-          setToasts(toasts.filter((toast) => toast.id !== removedToast.id));
-        }}
-        toastLifeTimeMs={60000}
-      />
-
-      <EuiFlexGroup
-        alignItems="center"
-        gutterSize="m"
-        style={{ maxWidth: 1316 }}
+      <PageHeader
+        appRightControls={getUseUpdatedUx() ? rightControls : []}
+        appBadgeControls={getUseUpdatedUx() ? badgeControls : []}
       >
-        <EuiFlexItem grow={false}>
-          <EuiFlexGroup gutterSize="m" alignItems="flexEnd">
-            <EuiFlexItem grow={false}>
-              <EuiTitle size="l">
-                <h1>{channel?.name || '-'}</h1>
-              </EuiTitle>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false} style={{ paddingBottom: 5 }}>
-              {channel?.is_enabled === undefined ? null : channel.is_enabled ? (
-                <EuiHealth color="success">Active</EuiHealth>
-              ) : (
-                <EuiHealth color="subdued">Muted</EuiHealth>
-              )}
-            </EuiFlexItem>
+        {!getUseUpdatedUx() && (
```

Comment:
Don't need this check, Page header should only render these when the new flag is disabled

Reply:
addressed